scripts/test-check-false-positives.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+cleanup() {
+  r=$?
+
+  # Dump stack logs
+  elastic-package stack dump -v --output "build/elastic-stack-dump/check-${PACKAGE_UNDER_TEST:-${PACKAGE_TEST_TYPE:-httpjson_false_positive_asserts}}"
+
+  # Take down the stack
+  elastic-package stack down -v
+
+  # Clean used resources
+  for d in test/packages/${PACKAGE_TEST_TYPE:-false_positives}/${PACKAGE_UNDER_TEST:-httpjson_false_positive_asserts}/; do
+    (
+      cd $d
+      elastic-package clean -v
+    )
+  done
+
+  # This is a false positive scenario and tests that the test case failure is a success scenario  
+  if [ "${PACKAGE_TEST_TYPE:-false_positives}" == "false_positives" ] && [ "${PACKAGE_UNDER_TEST:-httpjson_false_positive_asserts}" == "httpjson_false_positive_asserts" ]; then
```

Comment:
Nice to have a test for false positive cases, thanks! Not needed to change it here by now, but would it be possible to reuse this script to be able to add more test packages?

Reply:
Yes. It should be possible to add more packages under `PACKAGE_TEST_TYPE` `false_positives`